package.json

```diff
@@ -42,15 +48,18 @@
     "yargs": "17.1.1"
   },
   "devDependencies": {
-    "@commitlint/cli": "12.1.4",
-    "@commitlint/config-conventional": "12.1.4",
+    "@commitlint/cli": "13.1.0",
+    "@commitlint/config-conventional": "13.1.0",
     "@types/mocha": "9.0.0",
     "@types/node": "15.12.2",
     "@types/shelljs": "0.8.9",
     "@types/yargs": "17.0.2",
     "chai": "4.3.4",
+    "commitizen": "^4.2.4",
```

Comment:
Could we use specific versions? For example `4.2.4`

Reply:
Sure